server/http.go

```diff
@@ -73,10 +73,16 @@ func (p *Plugin) ServeHTTP(c *plugin.Context, w http.ResponseWriter, r *http.Req
 	}
 
 	status, err := p.serveHTTP(c, w, r)
-	if err != nil {
-		p.API.LogError("ERROR: ", "Status", strconv.Itoa(status), "Error", err.Error(), "Host", r.Host, "RequestURI", r.RequestURI, "Method", r.Method, "query", r.URL.Query().Encode())
+	switch {
+	case err == nil && status == http.StatusOK:
+		p.API.LogDebug("OK: ", "Status", strconv.Itoa(status), "Path", r.URL.Path, "Method", r.Method, "query", r.URL.Query().Encode())
+	case status == 0:
+		p.API.LogDebug("Passed to another router: ", "Path", r.URL.Path, "Method", r.Method)
```

Comment:
When does this case happen?

Reply:
Line 205, after gorilla handles it. The plugin "router" paths return a non-0.